ui/components/tabs/tabs.go

```diff
@@ -32,7 +33,11 @@ func (m Model) Update(msg tea.Msg) (Model, tea.Cmd) {
 func (m Model) View(ctx *context.ProgramContext) string {
 	sectionTitles := make([]string, 0, len(m.sectionsConfigs))
 	for i, section := range m.sectionsConfigs {
-		title := section.Title
+		filteringIsOnIndicator := "  "
+		if m.sectionIsFiltering[i] {
+			filteringIsOnIndicator = " "
```

Comment:
I think this icon is redundant. You can see the `repo:owner/name` already in the search field and it doesn't convey too much on it's own, like it's not clear what a github folder icon means.